docs/samples/extension/README.md

```diff
@@ -0,0 +1,58 @@
+This sample shows how to extend the tekton dashboard using the dashboard extension.  The extensions should be in experimental repository (https://github.com/tektoncd/experimental)
+
+## Build sample extension image 
+
+```
+cd docs/samples/extension
+docker build -t sampleextension .
+```
+## Push the sample container image to the registory (like dockerhub)
+```
+docker login --  if necessary
+docker tag sampleextension:latest <image name>:<image tag>  -- docker hub case: <image name>: <your dockerhub id>/sampleextension
+docker push <image name>:<image tag>
+```
+
+## Deploy the sample extension
+
+Edit config tekton-extension-sample.yaml  -- replace "image name" and "image tag"
+
+```
+kubectl apply -f config -n <dashboard installed namespace>
+```
+
+## Restart dashboard and port forwarding
+
+### Restart tekton-dashboard pod
+```
+kubectl delete pod $(kubectl get pod -l app=tekton-dashboard -o name -n <dashboard installed namespace>) -n <dashboard installed namespace>
+```
+### Restart port forwading
+```
+kubectl port-forward $(kubectl get pod -l app=tekton-dashboard -o name -n <dashboard installed namespace>) 9097:9097 -n <dashboard installed namespace>
+```
+
+## Access to the dashboard REST API
+```
+http://localhost:9097/v1/extension
+```
+### Output:
+```
+[
+ {
+  "name": "dashboard-extension",
+  "url": "/extensions/sample",
+  "port": "3000",
+  "displayname": "tekton_dashboard_extension",
+  "bundlelocation": "/bundle"
+ }
+]
+```
+## Access to the extension
+```
+http://localhost:9097/extensions/sample
+```
+### Output:
+```
+Hello tekton dashboard!! Here is /extenstions/sample
```

Comment:
I've added a comment below - this should read '/extensions/sample' once the code is fixed

Reply:
I misunderstood.  Yes, I fix it.